src/main/app.ts

```diff
@@ -266,6 +305,36 @@ class JupyterApplication implements IApplication, IStatefulService {
             shell.openExternal('https://github.com/jupyterlab/jupyterlab-desktop/releases');
         });
 
+        ipcMain.on('select-python-path', (event) => {
+            dialog.showOpenDialog({
+                properties: ['openFile', 'showHiddenFiles', 'noResolveAliases'],
```

Comment:
- Can we add `defaultPath` pointing to the last recently used path (if any)? Changing between pyenv environments would be easier this was.
- Should we add `filters` to only show python executables? [Not sure of this one]

Reply:
I see, agreed on `defaultPath`. not sure about filters, since users might have aliases etc.